src/lib/style/components/_expandable_table_data.scss

```diff
@@ -0,0 +1,27 @@
+.ExpandableTableData {
+  max-width: 250px;
+  vertical-align: middle;
+
+  p {
+    text-overflow: ellipsis;
+    margin: 0px 5px;
+  }
+
+  &[aria-expanded='true'] {
```

Comment:
Maybe apply a css transition to this to smooth it out?

Reply:
I don't think that can be done in a simple way, since we're toggling styles that cannot be animated. We might be able to do some hacky stuff with `max-height`